src/server_manager/web_app/ui_components/outline-server-view.ts

```diff
@@ -908,14 +825,6 @@ export class ServerView extends DirMixin(PolymerElement) {
     }
   }
 
-  // Help bubbles should be shown after this outline-server-view
-  // is on the screen (e.g. selected in iron-pages). If help bubbles
-  // are initialized before this point, setPosition will not work and
-  // they will appear in the top left of the view.
-  showGetConnectedHelpBubble() {
```

Comment:
Humm, we added this for a reason. I think it was to address the issue that managers weren't sure what to do next. I think it would be better to keep the bubble.

Reply:
Should I add into the regular access key row, so it'll be there for each instance?

Reply:
I guess so, but it should only ever show on key 0, if it wasn't dismissed before.